daemon/containerd/image_list.go

```diff
@@ -564,6 +568,9 @@ func setupLabelFilter(store content.Store, fltrs filters.Args) (func(image image
 			}
 			var cfg configLabels
 			if err := readConfig(ctx, store, desc, &cfg); err != nil {
+				if errdefs.IsNotFound(err) {
```

Comment:
Just to be sure; have we verified that this code works as expected? I noticed we're wrapping the error before checking the error-type. I _think_ that should still allow for both containerd's and moby's `errdefs` packages to detect the underlying error-type, but ... just in case (it's one of those cases where I'd blame myself not asking); https://github.com/moby/moby/blob/a9bca45e92c098205d378f59228af404e98bdb5e/daemon/containerd/image_list.go#L637-L641

Reply:
Yes, `errdefs.IsNotFound` uses `errors.Is` under the hood which will check the whole `Unwrap`'ed tree of errors.

Reply:
Thanks! Yes, I was _pretty_ sure it would work, but just in case 😅